Grammar/python.gram

```diff
@@ -632,6 +634,10 @@ invalid_comprehension:
 invalid_parameters:
     | param_no_default* (slash_with_default | param_with_default+) param_no_default {
         RAISE_SYNTAX_ERROR("non-default argument follows default argument") }
+invalid_star_etc:
+    | '*' (')' | ',' (')' | '**')) { RAISE_SYNTAX_ERROR("named arguments must follow bare *") }
+invalid_lambda_star_etc:
+    | '*' (':' | ',' (':' | '**')) { RAISE_SYNTAX_ERROR("named arguments must follow bare *") }
```

Comment:
Would it make sense to combine these two rules? Or would that be too weird?

Reply:
> Or would that be too weird?

Your call. IMHO it'd be a bit more difficult to understand a rule like that. "What's the colon doing there?"